operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/config/impl/AbstractSystemConfigurationImpl.java

```diff
@@ -72,7 +72,7 @@
 
     private static final byte[] NDJSON_LINESEPERATOR = "\r\n".getBytes();
 
-    public static final int IMPORT_NUMOFFHIRRESOURCES_PERREAD = 20;
+    public static final int IMPORT_NUMOFFHIRRESOURCES_PERREAD = 50;
```

Comment:
is this default config value documented anywhere?
not actually sure it needs to be, just trying to make sure we keep docs up-to-date

Reply:
It's always been hard coded. https://github.com/IBM/FHIR/blob/d5b8962ca2110afea41b4e4b78247be10e53c84d/operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/config/impl/AbstractSystemConfigurationImpl.java#L344 

20 had just felt low when I ran my tests (50 seems to be a decent chunk)

Reply:
ok, sounds good